src/components/Annotation/Annotation.js

```diff
@@ -71,44 +78,51 @@ class Annotation extends React.PureComponent<Props> {
         };
 
         return (
-            <div
-                className={classNames(`ba-annotation ${className}`, {
-                    'ba-is-pending': isPending || error
-                })}
-                onBlur={onBlur}
-                onFocus={onFocus}
-            >
-                <div className='ba-annotation-content'>
-                    <Avatar className='ba-annotation-avatar' {...createdBy} />
-                    <div className='ba-annotation-text'>
-                        <div className='ba-annotation-headline'>
-                            <UserLink className='ba-annotation-user-name' {...annotator} />
-                            <Tooltip
-                                text={
-                                    <FormattedMessage
-                                        {...messages.annotationPostedFullDateTime}
-                                        values={{ time: createdAtTimestamp }}
+            <Internationalize language={language} messages={intlMessages}>
+                <div
```

Comment:
Ideally, non-interactive elements shouldn't have event listeners. This currently can't be focused at all, right?

Reply:
onBlur and onFocus are used by the withFocus HOC